test/__tests__/Answer.test.js

```diff
@@ -120,7 +130,7 @@ describe("Answer component", () => {
     });
   });
 
-  it("loads typeahead options when layout class is typeahead and no possible values are specified", () => {
+  it("loads typeahead options when layout class is typeahead and no possible values are specified", async () => {
```

Comment:
From [react documentation](https://legacy.reactjs.org/docs/test-utils.html#act), a component must be prepared for assertion using [act()](https://legacy.reactjs.org/docs/test-utils.html#act). It is usually already integrated in [React-testing librabry](https://testing-library.com/docs/dom-testing-library/api-async/#waitfor), but in case of asynchronous updates, it must be wrapped in [waitFor()](https://davidwcai.medium.com/react-testing-library-and-the-not-wrapped-in-act-errors-491a5629193b)